client/gutenboarding/onboarding-block/index.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockConfiguration } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'automattic/onboarding';
+
+export enum SiteType {
+	BLOG = 'blog',
+	STORE = 'store',
+	STORY = 'story',
+}
+
+export interface Attributes {
+	align: 'full';
+	siteTitle: string;
+	siteType: undefined | SiteType;
```

Comment:
Equivalent? Fine to leave 🤷‍♂ 

```suggestion
	siteType?: SiteType;
```

Reply:
I like :+1: